### DISCUSSION TITLE
What version of  ghcr.io/goauthentik/ldap to choose

### DISCUSSION THREAD

**Discussion by francescor** [goauthentik/authentik#11585] [Q&A] [answered]
If in my docker-compose stack I use

```
  image: ghcr.io/goauthentik/server:2024.6.1
```

what version of the ldap outpost should I use? 

Currently I have 

```
    image: ghcr.io/goauthentik/ldap
```

and it works, but as a company's policy, I need to stick to a version (I cannot use `latest` images)

**Comment by BeryJu** [comment, marked as answer]
It is required to keep the authentik server/worker image on the same tag as any outpost (except during upgrades), so the image would be `ghcr.io/goauthentik/ldap:2024.6.1`

**Reply by francescor** [reply]
thank you so much!
